.github/workflows/containers.yaml

```diff
@@ -77,23 +75,27 @@ jobs:
           ${{ matrix.dockerfile_suffix }}:${tag} \
             --help
 
+      # ${{ github.repository_owner }}
```

Comment:
This seems superfluous now we're no longer using actor

Reply:
Dropped